src/Microsoft.Health.Fhir.Liquid.Converter/Ccda/CcdaTemplateProvider.cs

```diff
@@ -5,34 +5,20 @@
 
 using System.Collections.Generic;
 using DotLiquid;
-using DotLiquid.FileSystems;
-using Microsoft.Health.Fhir.Liquid.Converter.DotLiquids;
 using Microsoft.Health.Fhir.Liquid.Converter.Models;
 
 namespace Microsoft.Health.Fhir.Liquid.Converter.Ccda
 {
-    public class CcdaTemplateProvider : ITemplateProvider
+    public class CcdaTemplateProvider : TemplateProvider
```

Comment:
any difference between ccda provider and template provider?

Reply:
They are almost the same and can be easily joined into a unified class from engine's perspective. But since `Hl7v2TemplateProvider` and `CcdaTemplateProvider` are currently used in FHIR Server, I recommend that we keep these two classes at current stage.